vars/common-vars.yml

```diff
@@ -63,8 +63,8 @@ socok8s_deploy_config_location: "{{ socok8s_workspace }}"
 ucp_namespace_name: "{{ ucp_namespace | default('ucp') }}"
 
 # Variables used for building airship images locally
-suse_distro_identifier: opensuse_15
-suse_airship_components_base_image: "opensuse/leap:15.1"
+suse_distro_identifier: sle_15sp1
```

Comment:
We will need to be able to test at least both leap (from registry.opensuse.org ) and sle based images (from the internal registry attached to IBS, which is accessible from the CI that is in use here). Any idea how that will be done?